src/containers/PodcastSeries/components/PodcastEpisodes.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree. *
+ */
+
+import React from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import { FieldHeader } from '@ndla/forms';
+import { useFormikContext } from 'formik';
+import ElementList from '../../FormikForm/components/ElementList';
+import { AsyncDropdown } from '../../../components/Dropdown';
+import handleError from '../../../util/handleError';
+
+import {
+  AudioSearchResultType,
+  SeriesSearchResult,
+} from '../../../modules/audio/audioApiInterfaces';
+import { PodcastSeriesFormikType } from './PodcastSeriesForm';
+import { fetchAudio, searchAudio } from '../../../modules/audio/audioApi';
+
+const PodcastEpisodes = ({ t }: tType) => {
+  const {
+    values: { episodes, language },
+    setFieldValue,
+  } = useFormikContext<PodcastSeriesFormikType>();
+  const onAddEpisodeToList = async (audio: AudioSearchResultType) => {
+    try {
+      const newAudio = await fetchAudio(audio.id, language);
+      if (newAudio !== undefined) {
+        setFieldValue('episodes', [...episodes, newAudio]);
+      }
+    } catch (e) {
+      handleError(e);
+    }
+  };
+
+  const onUpdateElements = (eps: AudioSearchResultType[]) => {
+    setFieldValue('episodes', eps);
+  };
+
+  const searchForPodcasts = async (input: string): Promise<SeriesSearchResult> => {
+    return searchAudio({
+      query: input,
+      language: language,
+      audioType: 'podcast',
```

Comment:
Audiotype må være samme formatet som apiet tar imot. Det vil løse problemet med lyder som legges inn i serien.
```suggestion
      'audio-type': 'podcast',
```

Reply:
Ah, trodde jeg testet dette, men jeg søkte kanskje etter podcast når jeg testet :+1: